part-04-pydantic-schemas/app/main.py

```diff
@@ -37,7 +37,7 @@ def fetch_recipe(*, recipe_id: int) -> dict:
 @api_router.get("/search/", status_code=200, response_model=RecipeSearchResults)
 def search_recipes(
     *,
-    keyword: Optional[str] = Query(None, min_length=3, example="chicken"),
```

Comment:
@jncabelin this `examples` field isn't populating the swagger docs. Could you investigate please? Possibly wants `openapi_examples` instead